### ISSUE TITLE
SAVE/RESTORE commands broken

### ISSUE THREAD

**Issue by jestin** [theelkmechanic/ziggurat#2]
Just by typing the RESTORE command, the program crashes with this error:
![image](https://user-images.githubusercontent.com/669174/108931171-0de12780-75fc-11eb-8094-a2d8ce36dfd4.png)

When trying to run the SAVE command, the program crashes with this error:
![image](https://user-images.githubusercontent.com/669174/108931221-22252480-75fc-11eb-86e5-355a07d5e174.png)

I don't believe these are referring to 65c02 op-codes, since those would just be LDY a,y and LDA a,x which are both supported as far back as the 6502.  I suppose these are z-machine op-codes that the error is referring to, but I'm not familiar enough to know for sure.

**Comment by theelkmechanic** [comment]
Yep, that's the "I haven't implemented that Z-machine opcode yet" error. It's on the todo list. :)

**Comment by jestin** [comment]
Ah, no worries.  Thought it might have been a regression or something after the recent changes.  It's all looking great, by the way!
